hdat/hdat_cli.py

```diff
@@ -86,6 +99,10 @@ def hdat_cli(arguments, suites, golden_store, archive, git_info):
         results = resolve_resultspecs(archive, args.resultspec)
         for result in results:
             golden_store.insert(result)
+    elif args.command == 'csv':
+        results = resolve_resultspecs(archive, args.resultspec)
+        for result in results:
```

Comment:
The csv output is still showing a separate header for each result; the results need to handled altogether to get the common list of headers (e.g. `print_results(results, args.keys)` instead of `print_result`)